configuration/pipeline_config.json

```diff
@@ -7,6 +7,7 @@
       "TokenFileURL": "gs://avf-credentials/covid19-text-it-token.txt",
       "ContactsFileName": "covid19_contacts",
       "ActivationFlowNames": [
+        "covid19_s01e01_activation",
```

Comment:
just to check are they all in the same textit instance?

Reply:
Yes, these are just in the COVID19 instance, thanks.

To recap what's happened, because it is hard to follow:
 - We ran 4 weeks of nationwide 'pilot' programming asking people their thoughts on coronavirus, analysed by Project-COVID19.
- 2 weeks into the above, we started running the ke-urban shows, analysed by Project-COVID19-KE-Urban
- At the end of the 4 weeks of pilot programming, we harmonised the content with the ke-urban shows and pulled all data except the pilot into this pipeline
- We decided we wanted to analyse all of the Kenyan data as a whole, so are now including that here too.

Reply:
Great thanks